src/app/public/styles/_switch.scss

```diff
@@ -60,6 +59,7 @@ $sky-switch-size: 22px !default;
   height: $sky-switch-size;
   flex: 1 0 $sky-switch-size;
   margin: auto;
+  margin-top: 0;
```

Comment:
We've got some overlapping margin styles on these few lines. How about we roll them together in some shorthand: 
`margin: 0 5px auto auto;`

Reply:
Good call. Done